### DISCUSSION TITLE
File picker shows every file on the file system and is extremely slow to open

### DISCUSSION THREAD

**Discussion by AddictArts** [helix-editor/helix#4019] [Q&A] [answered]
Is there anyway to make it work like a normal file viewer, now all files, but folder based?

**Comment by dead10ck** [comment, marked as answer]
See https://github.com/helix-editor/helix/issues/1707 and https://github.com/helix-editor/helix/issues/200

**Reply by AddictArts** [reply]
Thanks, this is unfortunate. Example editing a .zshrc in your home dir, file picker is really pointless or un-usable. This behavior makes sense in a small project, but anything big it is just noise, lots of noise and slow, very very slow.

I am not trying to down on Helix. But, when evaluating a new editor and tool chain, I'm not in the context of a project and arguably a small one at that. I tend to edit a conf here or there. Open a single language file and check out how it works with it. The behavior is surprising and not in a good way.

**Reply by dead10ck** [reply]
Yeah, I would like it too. There is an open PR that implements it https://github.com/helix-editor/helix/pull/2377
